function/fetchFile.js

```diff
@@ -46,6 +46,7 @@ function makePrediction(file,url) {
       headers:{
         'Content-Type': 'application/json'
```

Comment:
This should be the content type of the image itself - so you'll need to detect that. Specifically:

* Call `getElementById` as you already do
* Get the `type` attribute of the file

In the interim you _might_ be able to leave it empty and let the browser determine - we can try that and go the more explicit path if we have to.

https://developers.google.com/web/fundamentals/media/capturing-images/#handling_a_filelist_object

Reply:
Okay! For now I'm calling type on file (which is obtained above in the code through .getElementById). If that doesn't work we can toy around with leaving that part empty